html/includes/table/devices.inc.php

```diff
@@ -203,11 +203,11 @@
             $col_port = '';
         }
         if ($port_count) {
-            $col_port = '<img src="images/icons/port.png" align="absmiddle"> ' . $port_count . '<br>';
+            $col_port = '<i class="fa fa-link" style="color:#0080FF"></i> ' . $port_count . '<br>';
```

Comment:
Its easy enough to change... I tried to match the blue icon that was originally used.